pygeoapi/processes/zonal_statistics_grass.py

```diff
@@ -63,15 +106,25 @@
                 'type': 'string'
             }
         },
-        'inputGeometries': {
-            'title': 'Input geometries',
-            'description': 'Input zones encoded as GeoJSON geometries',
+        'polygonGeoJson': {
+            'title': 'Polygon GeoJSON',
+            'description': 'A polygon GeoJSON for which to compute zonal statistics of the COG',  # noqa: E501
             'minOccurs': 1,
-            'maxOccurs': 'unbounded',
+            'maxOccurs': 1,
             'schema': {
-                '$ref': 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.json'  # noqa: E501
+                '$ref': 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.json'
+                # noqa: E501
```

Comment:
the comment should be after the line to take effect I guess

Reply:
fixed in [Fix lint](https://github.com/klips-project/klips-sdi/pull/144/commits/7740a49b5541ee67f5a10cf2c487995a010522d5)